account-kit/core/src/types.ts

```diff
@@ -56,15 +56,15 @@ export type AlchemyAccountsConfig = {
 export type Connection = {
   transport: AlchemyTransportConfig;
   chain: Chain;
-  policyId?: string;
+  policyId?: string | string[];
 };
 
 type RpcConnectionConfig =
```

Comment:
almost feels like we need one more permutation here like the following that allows you to specify the policyId array at the top level instead of nesting it

```ts
{
  chain: Chain;
  chains: {
    chain: Chain;
    transport?: AlchemyTransport
  }[],
  transport: AlchemyTransport;
  signerConnection?: ConnectionConfig;
  policyId?: string[];
}
| {
  chain: Chain;
  chains: {
    chain: Chain;
    transport?: AlchemyTransport
  }[],
  transport?: never;
  signerConnection: ConnectionConfig;
  policyId?: string[];
}
```